plugins/notifiers/slack/client.go

```diff
@@ -137,23 +143,21 @@ func (n *notifier) sendRequest(req *http.Request) (*userResponse, error) {
 }
 
 func parseMessage(message domain.NotificationMessage, templates domain.NotificationMessages) (string, error) {
-	var text string
-	switch message.Type {
-	case domain.NotificationTypeAccessRevoked:
-		text = templates.AccessRevoked
-	case domain.NotificationTypeAppealApproved:
-		text = templates.AppealApproved
-	case domain.NotificationTypeAppealRejected:
-		text = templates.AppealRejected
-	case domain.NotificationTypeApproverNotification:
-		text = templates.ApproverNotification
-	case domain.NotificationTypeExpirationReminder:
-		text = templates.ExpirationReminder
-	case domain.NotificationTypeOnBehalfAppealApproved:
-		text = templates.OthersAppealApproved
+	messageTypeTemplateMap := map[string]string{
+		domain.NotificationTypeAccessRevoked:          templates.AccessRevoked,
+		domain.NotificationTypeAppealApproved:         templates.AppealApproved,
+		domain.NotificationTypeAppealRejected:         templates.AppealRejected,
+		domain.NotificationTypeApproverNotification:   templates.ApproverNotification,
+		domain.NotificationTypeExpirationReminder:     templates.ExpirationReminder,
+		domain.NotificationTypeOnBehalfAppealApproved: templates.OthersAppealApproved,
+	}
+
+	block, ok := messageTypeTemplateMap[message.Type]
```

Comment:
can you change the variable name here too - `block` ?

Reply:
Done